index.html

```diff
@@ -52,6 +54,164 @@ <h1> Hey There. <br> I'm Dennis <br>
           </ul>
         </div>
       </section> 
+      
+      <section id="recentWorks">
+        <div class="imgHeader">
+          <h2>My Recent Works</h2>
+          <img src="./images/imgPlaceholder.png" alt="placeholderImg">
+        </div>
+        <div id="works">
+          <div class="introInfo">
+            <div class="title"><span>Multi-Post Stories</span></div>
```

Comment:
- [ ] Kindly I recommend to you use  a header tag `hx` the project title ,This way will be semantically correct and the screen readers and other technologies will thank you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

Reply:
header tag
Is there only one way of implementing header tags?

Reply:
Hello @til2to I recommend to use `hx` tag for  semantically correct and i try to give you clarification for suggested changes.